### DISCUSSION TITLE
请问2.0.0-rc.0应该如何解决markdown-it-katex中文解析失败（can't parse Chinese）

### DISCUSSION THREAD

**Discussion by wenleiwang** [vuepress/core#1437] [Q&A] [answered]
我在升级为2.0.0-rc.0后markdown-it-katex中文解析失败，我应该如何解决。如下是效果图：
![image](https://github.com/vuepress/vuepress-next/assets/39472071/01e46b18-1973-4241-b6a2-4f511181c3fc)

我的配置：

```
plugins: [
    mdEnhancePlugin({
        // 使用 KaTeX 启用 TeX 支持
        katex: true,
        // 使用 mathjax 启用 TeX 支持
        mathjax: true,
    }),
]
bundler: viteBundler({
    vuePluginOptions: {
        template: {
            compilerOptions: {
                isCustomElement: tag => customElement.includes(tag)
            }
        }
    }
}),
```

在^2.0.0-beta.67 版本是正常的如下是^2.0.0-beta.67版本图片：
![image](https://github.com/vuepress/vuepress-next/assets/39472071/3c4d107e-88b4-4882-bc50-1ea03da17e02)
是版本问题嘛，还是我少配置了什么？

---
I upgrade '2.0.0-rc.0'  find my 'markdown-it-katex' can't parse Chinese. 

>  Error show below:
> ParseError: KaTeX parse error: Expected '}', got '至' at position 18: 0^{308} \text{至̲} 10^{308}

is version question or my config lose some param？

please

**Comment by Mister-Hope** [comment, marked as answer]
Your title says `markdown-it-katex` has issues, while your code shows that you are using `vuepres-plugin-md-enhance`, which requires on `@mdit/plugin-katex`. First, please double check what you are doing and go to the correct place to report your problem with a valid minimal reproduction.

---

As the developer of plugin-md-enhance, I check the feature for you: 

<img width="243" alt="image" src="https://github.com/vuepress/vuepress-next/assets/33315834/4fcb74f0-50fb-4fa7-92a3-7a531fdb771d">
<img width="469" alt="image" src="https://github.com/vuepress/vuepress-next/assets/33315834/83e30ae7-fdbc-4e3b-ab66-6602e035916b">

everything is working as expected.

**Reply by wenleiwang** [reply]
Thank you very much. Yes, I mixed up `markdown-it-katex` and `vuepres-plugin-md-enhance`.
谢谢你，我弄混了 `markdown-it-katex`  和`vuepres-plugin-md-enhance`。

卸载掉`markdown-it-katex` 并安装`katex` 得到解决。
I remove `markdown-it-katex` and install `katex` . the problem is solved. Thank you again.
